packages/plugins/integration-zapier/src/lib/zapier-auth-code.service.ts

```
@@ -0,0 +1,70 @@
+    }> = new Map();
+
+    /**
+     * Generates and stores an authentication code for a user
+     *
+     * @param userId The user's ID
+     * @param tenantId The tenant ID
+     * @param organizationId The organization ID
+     * @returns The generated authorization code
+     */
+
+    generateAuthCode(userId: ID, tenantId: ID, organizationId?: ID): String {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)